alias.config.js

```diff
@@ -0,0 +1,15 @@
+import path from 'path';
+
+export default {
+  /**
+   * Reminder to ensure aliases are added to the following:
+   * 1. .storybook/main.js.
+   * 2. tsconfig.json
+   */
+  $components: path.resolve('./src/lib/components'),
+  $journey: path.resolve('./src/lib/journey'),
+  $lib: path.resolve('./src/lib'),
+  $locales: path.resolve('./src/locales'),
+  $package: path.resolve('./package'),
+  $widget: path.resolve('./src/widget'),
```

Comment:
Moved all this to a reusable config. Unfortunately, we still have to manually add new aliases to `tsconfig.json` (because it's JSON) and `.storybook/main.js` (because this is CJS).